### PR TITLE
docs: clearer instructions

### PR DESCRIPTION
**Pull request by Ansonhkg**
# What

- The package tables have been restructured into two distinct sections. The first section highlights our primary entry points, `lit-node-client` and `lit-node-client-nodejs`. The second part is tailored for technically adept users, listing additional modules that they may find useful. (see [here](https://github.com/LIT-Protocol/js-sdk/tree/feature/lit-767-js-sdk-update-docs-to-hidetoggle-all-other-available#packages)) 
- Added a new `Quick Start` guide in the Contributing section offering straightforward steps to getting started (see [here](https://github.com/LIT-Protocol/js-sdk/tree/feature/lit-767-js-sdk-update-docs-to-hidetoggle-all-other-available#contributing-and-developing-to-this-sdk))

For anyone who's interested, [here](https://github.com/LIT-Protocol/js-sdk/blob/a53ec30fe1a80f4104864b210f430964e8d34c52/tools/scripts/gen-readme.mjs#L60) how the tables are generated

### CODE REVIEW

**Review by sarahzdev — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -55,16 +63,21 @@ yarn add @lit-protocol/lit-node-client
 
 # Packages
 
-> 📝 If you're looking to use the Lit SDK, you're probably all set with just the lit-node-client <link>. It's got everything most folks need. However, if you're the type who likes to keep things super lean and mean, and you need to keep an eagle eye on your bundle size, we've got individual packages too.
-
-Individual packages are below so that you can import the minimum required packages for your use case.
+📝 If you're looking to use the Lit SDK, you're probably all set with just the lit-node-client <link>. It's got everything most folks need.
```
Comment (sarahzdev) [inline README.md:66]: maybe rephrase the sentence along the lines of "get started with interacting with lit network"

**Comment by Ansonhkg** [comment]
@sarahzdev updated!
